src/default_config_params/mod.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+pub fn get_default_wallet_path() -> PathBuf {
+    let home_var = std::env::var("HOME");
+    match home_var {
+        Ok(home_path) => {
+            let teleport_config_dir = PathBuf::from(home_path).join(".teleport");
+            if teleport_config_dir.exists() {
+                let wallet_path = teleport_config_dir.join("wallet.teleport");
+
+                wallet_path
+            } else {
+                std::fs::create_dir_all(teleport_config_dir.clone()).unwrap();
+                let wallet_path = teleport_config_dir.join("wallet.teleport");
+
+                wallet_path
+            }
+        }
+        Err(e) => {
+            println!("HOME env variable not set on this Terminal instance. {e}. Creating wallet in default project path");
+
+            let wallet_path = PathBuf::from("wallet.teleport");
```

Comment:
This ain't the default path.

Reply:
In cases where we are unable to determine the home_dir for the system, what should be the default path? In the above, I'd assumed the application's current location.